ovs/lib/setup.py

```diff
@@ -258,33 +257,160 @@ def setup_node(node_type=None):
                 if avahi_installed is True and cluster_name is None:
                     raise RuntimeError('The name of the cluster should be known by now.')
 
-                node_password = master_password
-                for node_info in SetupController.nodes.itervalues():
+                ip_hostname_map = {cluster_ip: list({node_name, fqdn_name})}
+                for node_host_name, node_info in SetupController.nodes.iteritems():
```

Comment:
Actually i do need to loop, because i don't know the hostname of the master node
And i need the hostname to put in the ip_hostname_map
--> will still add a break in if statement